alacritty/src/renderer/text/builtin_font.rs

```diff
@@ -879,24 +879,27 @@ impl Canvas {
     /// `self.width` and an attached rectangle to form a "╭" using a given `stroke_size` in the
     /// bottom-right quadrant of the `Canvas` coordinate system.
     fn draw_rounded_corner(&mut self, stroke_size: usize) {
-        let r = 0.5 * (self.width.min(self.height) + stroke_size) as f32;
+        let r = (self.width.min(self.height) + stroke_size) as f32 / 2.;
         let w = stroke_size as f32;
```

Comment:
`w` is not really a clear name, since it's not clear what width it is, would just suggest to use `let stroke_size = stroke_size as f32` for that purpose.